core/print_api.php

```diff
@@ -2095,3 +2095,19 @@ function print_max_filesize( $p_size, $p_divider = 1000, $p_unit = 'kb' ) {
 		. get_filesize_info( $p_size / $p_divider, lang_get( $p_unit ) );
 	echo '</span>';
 }
+
+/**
+ * Print a button which presents a standalone form.
+ * This function remains for compatibility with v1.3
+ * @deprecated use print_form_button() instead
+ * @param string $p_action_page    The action page.
+ * @param string $p_label          The button label.
+ * @param array  $p_args_to_post   Associative array of arguments to be posted
+ * @param mixed  $p_security_token Optional; null (default), OFF or security token string.
+ * @see form_security_token()
+ * @see print_form_button()
+ * @return void
+ */
+function print_button( $p_action_page, $p_label, array $p_args_to_post = null, $p_security_token = null ) {
+	print_form_button( $p_action_page, $p_label, $p_args_to_post, $p_security_token );
```

Comment:
Since the function is deprectated, shouldn't it trigger a E_USER_DEPRECATED error ?